package.json

```diff
@@ -1,23 +1,22 @@
 {
-  "name": "monowingo",
+  "name": "@winglang/monorepo",
+  "private": true,
   "devDependencies": {
-    "@nrwl/devkit": "16.5.0",
     "bump-pack": "workspace:^",
     "changelogen": "^0.5.3",
-    "nx": "16.5.0",
-    "semver": "^7.3.8"
+    "semver": "^7.3.8",
+    "turbo": "^1.10.7"
   },
   "scripts": {
-    "build": "nx run-many --all --targets build,test,package",
-    "test": "nx run-many --all --target=test",
-    "package": "nx run-many --all --target=package",
-    "build:ci": "nx run-many --targets lint,build --all --configuration=release --output-style=stream --verbose",
-    "test:ci": "nx run-many --exclude=hangar --target=test --all --configuration=release --output-style=stream --verbose",
-    "package:ci": "nx run-many --target=package --all --configuration=release --output-style=stream --verbose",
+    "compile": "turbo compile --filter=!hangar",
```

Comment:
Not sure we want a "compile" command at the root

Reply:
Removed.